x-pack/test_serverless/functional/test_suites/common/console/console.ts

```diff
@@ -191,27 +197,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.header.waitUntilLoadingHasFinished();
         expect(await PageObjects.console.hasFolds()).to.be(false);
       });
-
```

Comment:
I'm open to suggestions for what other tests here are redundant. I was thinking about removing everything from line 89 and below?

Reply:
Since we extensively test everything for stateless, I think the only thing we care about for serverless is that the console app loads and that it can send a command to the api and get a valid result back. So I'd say anything but that should be gone. wdy?

Reply:
Yes, this makes sense to me! Removed the redundant tests in [51bae3a](https://github.com/elastic/kibana/pull/172664/commits/51bae3a5885f33439351101c19b688cc0cd3bdca).